androidApp/src/main/java/com/pulselink/ui/screens/ContactDetailScreen.kt

```
@@ -248,6 +259,63 @@ private fun LinkStatusSection(
+                                        data = Uri.parse("mailto:")
+                                        putExtra(Intent.EXTRA_EMAIL, arrayOf(contact.email ?: ""))
+                                        putExtra(Intent.EXTRA_SUBJECT, "Override Instructions")
+                                        putExtra(
+                                            Intent.EXTRA_TEXT,
+                                            "You have been set as a trusted contact. Even without the PulseLink app, you can trigger an emergency alert on my phone by texting exactly:\n\n'pulselink ${contact.remotePin} emergency'\n\nto my number."
+                                        )
+                                    }
+                                    try {
+                                        context.startActivity(emailIntent)
+                                        Toast.makeText(context, "Opening email client...", Toast.LENGTH_SHORT).show()
+                                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)